lib/bolt/outputter.rb

```diff
@@ -9,6 +9,11 @@ def self.for_format(format, color, verbose, trace)
       when 'json'
         Bolt::Outputter::JSON.new(color, verbose, trace)
       when 'rainbow'
+        begin
+          require 'bolt/outputter/rainbow'
+        rescue LoadError
```

Comment:
I think it would be better to move `require 'paint'` into the initialize method of the Rainbow outputter and guard it there. That would be a little more constrained than guarding the selection+loading of the entire outputter.

Reply:
Specifically, it seems less prone to error to have the code fail when you try to _use_ it rather than fail when you try to _load_ it.

Reply:
For sure. I can move it there.